packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2188,6 +2188,13 @@ def get(self, id, authed_user_id):
     type=str,
     choices=SortDirection._member_names_,
 )
+purchases_and_sales_parser.add_argument(
+    "content_ids",
+    required=False,
+    description="Content IDs to filter by",
+    type=int,
```

Comment:
Arguably these should be hashIds I think. Do we use int IDs for other query params?

Reply:
good point